WindowsServerDocs/remote/remote-desktop-services/clients/change-listening-port.md

```diff
@@ -22,3 +22,26 @@ When you connect to a computer (either a Windows client or Windows Server) throu
 6. Close the registry editor, and restart your computer.
 
 The next time you connect to this computer by using the Remote Desktop connection, you must type the new port. If you're using a firewall, make sure to configure your firewall to permit connections to the new port number.
+
+
+You can always check the current port through PowerShell too with below commands:
+
+~~~powershell
+Get-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Terminal Server\WinStations\RDP-Tcp' -name "PortNumber"
+~~~
+Example:
+~~~powershell
+PortNumber   : 3389
+PSPath       : Microsoft.PowerShell.Core\Registry::HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Terminal Server\WinStations\RDP-Tcp
+PSParentPath : Microsoft.PowerShell.Core\Registry::HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Terminal Server\WinStations
+PSChildName  : RDP-Tcp
+PSDrive      : HKLM
+PSProvider   : Microsoft.PowerShell.Core\Registry
+~~~
```

Comment:
Yes, that is sort of what I tried to suggest, but using the GitHub comment suggestion feature (Ctrl-G), it is quite difficult since the suggestion feature uses backticks itself as a suggestion code block fence.

@vamsivellanki : Would you be able to replace all the tilde characters with backticks, as suggested?

Reply:
~~~suggestion
```
~~~